src/robusta/patch/patch.py

```diff
@@ -0,0 +1,122 @@
+from typing import Union, List, Dict, get_type_hints
+from dataclasses import is_dataclass, InitVar
+from inspect import signature, getmodule
+from hikaru import HikaruDocumentBase, HikaruBase
+from ruamel.yaml import YAML
+
+try:
+    from typing import get_args, get_origin
+except ImportError:  # pragma: no cover
+    def get_args(tp):
```

Comment:
This is weird. I'm sure there is a good reason for it though! Perhaps add a comment explaining why we're doing this?

Reply:
I'm not sure tbh. I just copied that section from Hikaru meta.py
Do you think we should change something here?

Reply:
No it's fine. Probably just to support older python versions